aspnetcore/razor-components/get-started.md

```diff
@@ -5,58 +5,162 @@ description: Learn how to get started with the Razor Components framework.
+
+1. Add a *launch.json* file to the *.vscode* folder with the following content:
+
+   [!code-json[](get-started/samples_snapshot/3.x/launch.json)]
+
+1. Execute the app using the Visual Studio Code debugger.
+
+1. In a browser, navigate to `https://localhost:5001`.
+
+Congrats! You just ran your first Razor Components app!
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+.NET Core 3.0 will be supported with Visual Studio for Mac version 8.0 or later. Visual Studio for Mac version 8.0 Preview isn't available at this time.
+
+Use the [.NET Core CLI version of this topic](xref:razor-components/get-started?tabs=netcore-cli) on macOS.
+
+
+[!INCLUDE[](~/includes/net-core-prereqs-mac-3.0.md)]
 
-   ![New app dialog](https://msdnshared.blob.core.windows.net/media/2018/07/new-blazor-app-dialog-0.5.0.png)
+To create your first project Razor Components project in Visual Studio for Mac:
 
-1. Press **Ctrl-F5** to run the app *without the debugger*. Running with the debugger (**F5**) isn't supported at this time.
+1. Select **File** > **New Solution** or **New Project**.
+1. In the sidebar, select **.NET Core** > **App**.
+1. Select **ASP.NET Core Razor Components** and select **Next**.
+1. The **Target Framework** defaults to **.NET Core 3.0**. Select **Next**.
+1. In the **Project Name** field, enter `WebApplication1`. Select **Create**.
+1. Select **Run** > **Run Without Debugging** to run the app *without the debugger*. Running with the debugger isn't supported at this time.
 
-To create a new Blazor app from the command-line:
+Congratulations! You just ran your first Razor Components app!
+-->
+
+# [.NET Core CLI](#tab/netcore-cli/)
+
+Prerequisites:
+
+* [.NET Core SDK 3.0 Preview](https://dotnet.microsoft.com/download/dotnet-core/3.0)
+
+To create your first Razor Components project from a command shell:
 
 ```console
-dotnet new blazor -o BlazorApp1
-cd BlazorApp1
+dotnet new razorcomponents -o WebApplication1
+cd WebApplication1
 dotnet run
 ```
 
-Congrats! You just ran your first Blazor app!
+Congratulations! You just ran your first Razor Components app!
+
+---
+
+## Razor Components project
+
+The solution created by the Razor Components template contains two projects:
+
+* *WebApplication1.Server* &ndash; The server project is an ASP.NET Core project set up to host the Razor Components app.
+* *WebApplication1.App* &ndash; The client-side web UI project that uses Razor Components.
+
+The UI logic in the *WebApplication1.App* project is separated from the rest of the app due to a technical limitation in ASP.NET Core 3.0 Preview 2. The Razor file extension (*.cshtml*) used for Razor Components is also used for Razor Pages and MVC views. Currently, Razor Components and Razor Pages/MVC have different compilation models, so the Razor Components Razor files are kept separate. In a future preview, we plan to introduce a new file extension for Razor Components (*.razor*). Components, pages, and views will be hosted *in the same project*.
```

Comment:
Have you already created an issue to revisit this first sentence when the next release it out?

Reply:
https://github.com/aspnet/Docs/issues/10717